piecewise_web/css/mlab.css

```diff
@@ -612,7 +612,7 @@ a.leaflet-popup-close-button {
     margin-top: 1em;
   }
   #sidebar h3 {
-    font-size: 2.5em;
+    /* font-size: 2.5em; */
```

Comment:
You could leave this here commented out, but I think it's our preference to always remove things rather than commenting them out to keep the code as clean as possible, unless this is just temporary and you think you're going to add it back shortly.